tests/Simpleue/Mocks/JobSpy.php

```diff
@@ -11,12 +11,18 @@
 class JobSpy implements Job {
 
     private $manageCounter;
+    private $quitCount;
+    private $testSignal;
 
     public function _construct() {
         $this->manageCounter = 0;
     }
 
     public function manage($job) {
+        if ($this->quitCount && (($this->manageCounter+1) === $this->quitCount)) {
```

Comment:
I think the point of doing it before incrementing is so we can be sure that the manage method continues once more before the worker exits.